### PR TITLE
BZ-2100180 Adding known issue for metallb upgrade

### PR DESCRIPTION
**Pull request by kquinn1204**
[BZ#2100180]: Upgrading the MetalLB Operator to 4.11 version fails.

Version(s):
Known issue for 4.11 RN

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2100180

Link to docs preview: http://file.emea.redhat.com/kquinn/BZ-2100180/release_notes/ocp-4-11-release-notes.html#ocp-4-11-known-issues
NOTE:
Cross ref dependency on https://github.com/openshift/openshift-docs/pull/45516 merged.

### CODE REVIEW

**Comment by tmulquee** [comment]
Preview available at http://file.emea.redhat.com/tmulquee/TELCODOCS-751/release_notes/ocp-4-11-release-notes.html#ocp-4-11-known-issues

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -1102,6 +1102,12 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 * In the {rh-openstack-first}, a potential issue can affect Kuryr when the ports pools are populated with a variety of bulk port creation requests made at the same time. During these bulk requests, if the IP allocation for one of the IP addresses fails, the Neutron retries the operation for all ports. This issue was resolved in a previous errata; however, Red Hat suggests setting a small value for the ports pools batch to avoid large bulk port creation requests. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024690[*BZ2024690*])
 
+* In 4.11 the MetalLB Operator scope has changed from namespace to cluster and this results in upgrading from a prior version failing. 
++
+Workaround: Remove the previous version of the MetalLB Operator. Do not delete the namespace or the instance of the MetalLB custom resource, and deploy the new Operator version. This keeps metallb running and configured.
```
[the remark this answers is not in the document]
Comment (ahardin-rh) [reply release_notes/ocp-4-11-release-notes.adoc:1107]: s/metallb/MetalLB

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -1102,6 +1102,12 @@ This script removes unauthenticated subjects from the following cluster role bin
...
++
+For more information, see xref:../networking/metallb/metalb-upgrading-operator.adoc#upgrading-metallb-operator_metallb-operator-upgrade [Upgrading the MetalLB Operator]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100180[*BZ#2100180*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-11-release-notes.adoc:1109]: Looks like this link is dependent on your other PR so the build is failing

**Comment by ahardin-rh** [comment]
Adding Peer Review Done label, but acknowledging that the checks are currently failing until the related PR this depends on merges.

**Review by pneedle-rh — COMMENTED** [review]
@kquinn1204 I have completed my review of this PR and have added one minor suggestion.

Please also squash your commits into one.

For my review, I cloned your feature branch and built a local preview. If you get a chance, please add a link to a preview in the PR description after you address the suggestion.

Aside from these points, the PR looks good to me. Thanks!

**Pull request merged**